services/dummy-test-service/main.go

```diff
@@ -1,7 +1,295 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+/*type user struct {
```

Comment:
Don't place commented out code in the commits.
If you want to have a "local stash" for commented code you can:
1. create dir (e.g., ./tmp/)
2. create .gitignore in dir (e.g., ./tmp/.gitignore)
3. place wild card on .gitignore (i.e., *)
4. place your "tmp" files there

Reply:
it was just in case all i implemented suddenly needed to be backtracked. i will remove